src/main/java/org/sonarlint/cli/report/ResourceReport.java

```diff
@@ -61,7 +62,7 @@ public String getName() {
   }
 
   public String getType() {
-    if (filePath == null) {
+    if (filePath == null || filePath == Paths.get("")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this comparison to use the equals method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1698)